transaction-status/Cargo.toml

```diff
@@ -28,6 +28,8 @@ spl-associated-token-account = { workspace = true, features = ["no-entrypoint"]
 spl-memo = { workspace = true, features = ["no-entrypoint"] }
 spl-token = { workspace = true, features = ["no-entrypoint"] }
 spl-token-2022 = { workspace = true, features = ["no-entrypoint"] }
+spl-token-group-interface = { workspace = true }
+spl-token-metadata-interface = { workspace = true, features = ["serde-traits"] }
```

Comment:
I'm only just now realizing that we don't use `"serde-traits"` for token-2022. It would probably have saved us a ton of work :sweat:

Reply:
RIP. 🪦